lib/inspec/profile_context.rb

```diff
@@ -67,18 +68,31 @@ def profile_supports_os?
 
     def all_rules
```

Comment:
we should rename this to `all_controls` and provide an alias for backwards-compatibility